src/lib/Link.svelte

```diff
@@ -52,6 +52,15 @@
 
   // this will enable `<Link on:click={...} />` calls
   function onClick(e) {
+    // user used a keyboard shortcut to force open link in a new tab
+    if (e.target.tagName === 'A' || e.target.tagName === 'BUTTON') {
```

Comment:
It is possible to middle-click a button that opens in a new tab? If that's true then the first-level `if()` is not longer needed.

Reply:
ya know I was just following the lead based on what was presented with Link and my own usage.. maybe I was thinking about it wrong.. let me find a few sites and do a bit more homework.

Reply:
ok, I think you're correct, nice catch.. I was just looking through the different things on github and true buttons don't behave like that... let me update.